experimental/LieAlgebras/src/simple_lie_algebra.jl

```diff
@@ -0,0 +1,174 @@
+    #note that it is enough to do this over QQ, as we can later coerce the constants
+    #into the field R
+    coeffs_iso = inv(Oscar.iso_oscar_gap(QQ))
+    LG = GAP.Globals.SimpleLieAlgebra(GAP.Obj(S), n, domain(coeffs_iso))
+    sc_table_G =
+      (
+        entry -> (entry[1], Vector{elem_type(QQ)}(map(coeffs_iso, entry[2])))
+      ).(
+        Matrix{Tuple{Vector{Int},Vector{GAP.Obj}}}(
+          (GAP.Globals.StructureConstantsTable(GAPWrap.Basis(LG)))[1:dimL]
+        )
+      )
+    struct_consts = Matrix{SRow{elem_type(R)}}(undef, dimL, dimL)
+    for i in 1:dimL, j in 1:dimL
+      struct_consts[i, j] = sparse_row(
+        R, Tuple{Int,elem_type(R)}[(k, R(c)) for (k, c) in zip(sc_table_G[i, j]...)]
+      )
+    end
+    return get_cached!(SimpleLieAlgebraDict, (R, RS), cached) do
+      new{C}(R, RS, dimL, s, st, struct_consts)
+    end::SimpleLieAlgebra{C}
+  end
+end
+
+const SimpleLieAlgebraDict = CacheDictType{Tuple{Field,RootSystem},SimpleLieAlgebra}()
+
+mutable struct SimpleLieAlgebraElem{C<:FieldElem} <: LieAlgebraElem{C}
+  parent::SimpleLieAlgebra{C}
+  mat::MatElem{C}
+end
+
+###############################################################################
+#
+#   Basic manipulation
+#
+###############################################################################
+
+parent_type(::Type{SimpleLieAlgebraElem{C}}) where {C<:FieldElem} = SimpleLieAlgebra{C}
+
+elem_type(::Type{SimpleLieAlgebra{C}}) where {C<:FieldElem} = SimpleLieAlgebraElem{C}
+
+parent(x::SimpleLieAlgebraElem) = x.parent
+
+base_ring(L::SimpleLieAlgebra{C}) where {C<:FieldElem} = L.R::parent_type(C)
+
+coefficient_ring(L::SimpleLieAlgebra{C}) where {C<:FieldElem} = L.R::parent_type(C)
+
+dim(L::SimpleLieAlgebra) = L.dim
+
+root_system(L::SimpleLieAlgebra) = L.root_system
+
+root_system_type(L::SimpleLieAlgebra) = L.root_system_type
+
+#  Base.@propagate_inbounds function Base.setindex!(f::SimpleLieAlgebraElem{T}, d, r) where {T <: FieldElem}
+#    Generic._matrix(f)[1, r] = d
+#    return f
+#  end
+
+function symbols(L::SimpleLieAlgebra)
+  return L.s
+end
+
+###############################################################################
+#
+#   String I/O
+#
+###############################################################################
+#make root system string
+function string_root_system(L::SimpleLieAlgebra)
+  return string(root_system_type(L)[1]) * string(root_system_type(L)[2])
+end
+
+function Base.show(io::IO, ::MIME"text/plain", L::SimpleLieAlgebra)
+  io = pretty(io)
+  println(io, "Simple Lie algebra")
+  println(
+    io,
+    Indent(),
+    "of type $(string(root_system_type(L)[1])*string(root_system_type(L)[2]))",
+    Dedent(),
+  )
+  println(io, Indent(), "of dimension $(dim(L))", Dedent())
+  print(io, "over ")
+  print(io, Lowercase(), base_ring(L))
+end
+
+function Base.show(io::IO, L::SimpleLieAlgebra)
+  if get(io, :supercompact, false)
+    print(io, "Simple Lie algebra")
+    print(io, "$(string_root_system(L))")
```

Comment:
```suggestion
    print(io, "Simple Lie algebra $(root_system_type_string(root_system(L))")
```
simplify using print helper for root systems